css/style.css

```
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
+  padding: 1rem 0 1rem 0;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+/* Navigation */
+.company-logo {
+  padding-left: 0.8rem;
+  width: 2.3rem;
+  height: 2.3rem;
+}
+
+.hamburger-icon {
+  padding: 0.8rem 0.8rem 0 0;
+  width: 1.5rem;
+  height: 1.1rem;
+}
+
+.navbar {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.navbar-hamburger-Menu {
+  display: flex;
+  flex-flow: column nowrap;
+  background-color: #fff;
+  border: 0;
+}
+
+.navbar-full-Menu.show {
+  display: none;
+}
+
+.navbar-full-Menu {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.navbar-full-Menu > li {
+  list-style-type: none;
+}
+
+.navbar-full-Menu > li:first-child {
+  color: #4b4b4d;
+  font-weight: 500;
+}
+
+.navbar-full-Menu > li > a {
+  line-height: 3rem;
+  color: #89899e;
+  text-decoration: none;
+}
+
+.navbar-full-Menu > li > a:hover {
```

```suggestion
.navbar-full-Menu > li > a:focus,
.navbar-full-Menu > li > a:hover {
```
When adding hover, we should typically add focus too so keyboard users pressing the `Tab` key to jump to the button get this style too

---

This is an oversight, I realise its good practice, thanks for pointing it out :+1: